modules/features/perf/opencl/perf_feature2d.cpp

```diff
@@ -7,7 +7,7 @@
 namespace opencv_test {
 namespace ocl {
 
-OCL_PERF_TEST_P(feature2d, detect, testing::Combine(Feature2DType::all(), TEST_IMAGES))
```

Comment:
I propose to presume the original names to compare 4.x and 5.x reports. @vpisarev What do you think?

Reply:
Also the test rename triggers changes in performance regression files in opencv_extra.

Reply:
I think it's a good idea to keep names of the tests as "features2d", it will minimize the difference between 4.x and 5.x.